docs/src/increase-observability/metrics/dedicated-generation-3.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Monitor Dedicated Generation 3 metrics
+sidebarTitle: Dedicated Generation 3 metrics
+description: Understand how to read metrics for Dedicated Genration 3 environments.
+---
+
+Dedicated Generation 3 environments consist of:
+
+* App containers: one or more [app containers](../../create-apps/_index.md)
+* Service containers: zero or more [service containers](../../add-services/_index.md)
+* Worker containers: zero or more [worker instances](../../create-apps/app-reference.md#workers).
+
+Infrastructure metrics report CPU, RAM, and disk space for all containers.
+
+By default, the graphs include all hosts and an average over the hosts.
+To select metrics for specific hosts, click **Filter**.
+
+![Clicking Filter reveals a list of hosts you can filter](/images/metrics/filtering-gen3.png "0.4")
+
+The IDs for the hosts in the list for filtering match the IDs for interacting with a host,
```

Comment:
this is not totally accurate. When ssh'ing to a DG3 env, i see i get taken to:
web@app.0:~$

So the Host 0 matched app.0, same for Host 1 (app.1), Host 2(app.2).

Is it worth tweaking this in the docs @CollierCZ ?